src/components/datepicker/bl-datepicker.css

```diff
@@ -0,0 +1,54 @@
+:host {
+  width: fit-content;
+
+  --bl-input-cursor: pointer;
+  --icon-size: var(--line-height);
+  --icon-color: var(--bl-color-neutral-light);
+  --datepicker-width: 314px;
+
+  display: block;
```

Comment:
Everything you put here can be edited from outside. It would be good to delete anything that might interfere with the design system.